src/main/java/az/gdg/mscomplaint/service/ComplaintService.java

```diff
@@ -0,0 +1,18 @@
+package az.gdg.mscomplaint.service;
+
+import az.gdg.mscomplaint.model.ComplaintRequest;
+import az.gdg.mscomplaint.model.dto.ComplaintDTO;
+
+import java.util.List;
+
+public interface ComplaintService {
+
+    public List<ComplaintDTO> getAllComplaints();
+
+    public void createComplaint(ComplaintRequest complaintRequest);
+
+    public void updateComplaint(ComplaintDTO complaintDTO);
+
+    public void deleteComplaint(int id);
```

Comment:
It is an interface, you do not need to specify public access modifier, it is redundant.